docker/README.md

```diff
@@ -0,0 +1,117 @@
+Sending build context to Docker daemon  5.632kB
+Step 1/13 : FROM fedora:latest
+ ---> c582c1438f27
+...
+...
+Step 13/13 : RUN echo "cd /scratch" >> /root/.bashrc
+ ---> Running in 327a5e0e968d
+Removing intermediate container 327a5e0e968d
+ ---> 5befe28e30eb
+Successfully built 5befe28e30eb
+Successfully tagged ebplane_volume:latest
+```
+
+## Start the Docker container
+
+We can start the container in the background and connect to it whenever needed,
+or start an instance and use it for the time being. See the header in the
+[Dockerfile](Dockerfile) for some examples. Here we will start a container and
+use it interactively.
+
+```
+[host:~/...ratch/ebplane/docker]$ docker run -h ebplane -itv ~/Documents/Scratch:/scratch ebplane_volume bash
+[root@ebplane scratch]# 
+```
+
+## Now build and run the unit tests
+
+Now that we are attached to the Docker container we can build all the binaries:
+
+```
+[root@ebplane scratch]# cd ebplane
+[root@ebplane ebplane]# bazel build ...:all
+Extracting Bazel installation...
+Starting local Bazel server and connecting to it...
+DEBUG: /root/.cache/bazel/_bazel_root/4a41d10c950280d61cfeba75245d2362/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:9: rbe_default not using checked in configs; Bazel version 0.29.1 was picked/selected but no checked in config was found in map {"0.20.0": ["8.0.0"], "0.21.0": ["8.0.0"], "0.22.0": ["8.0.0", "9.0.0"], "0.23.0": ["8.0.0", "9.0.0"], "0.23.1": ["8.0.0", "9.0.0"], "0.23.2": ["9.0.0"], "0.24.0": ["9.0.0"], "0.24.1": ["9.0.0"], "0.25.0": ["9.0.0"], "0.25.1": ["9.0.0"], "0.25.2": ["9.0.0"], "0.26.0": ["9.0.0"], "0.26.1": ["9.0.0"], "0.27.0": ["9.0.0"], "0.27.1": ["9.0.0"], "0.28.0": ["9.0.0"], "0.28.1": ["9.0.0"]}
+INFO: Analyzed 31 targets (37 packages loaded, 67557 targets configured).
+INFO: Found 31 targets...
+INFO: From CcGnumakeMakeRule external/linuxsrc/installhdr/include:
+
+INFO: From CcConfigureMakeRule external/elfutils/libelf/include:
+
+INFO: From CcGnumakeMakeRule external/libbpf/libbpf/include:
+
+INFO: Elapsed time: 227.548s, Critical Path: 79.67s
+INFO: 74 processes: 74 processwrapper-sandbox.
+INFO: Build completed successfully, 146 total actions
+[root@ebplane ebplane]# 
+```
+
+And finish it off with running the unit tests:
+
+```
+[root@ebplane ebplane]# bazel test ...:all
+DEBUG: /root/.cache/bazel/_bazel_root/4a41d10c950280d61cfeba75245d2362/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:9: rbe_default not using checked in configs; Bazel version 0.29.1 was picked/selected but no checked in config was found in map {"0.20.0": ["8.0.0"], "0.21.0": ["8.0.0"], "0.22.0": ["8.0.0", "9.0.0"], "0.23.0": ["8.0.0", "9.0.0"], "0.23.1": ["8.0.0", "9.0.0"], "0.23.2": ["9.0.0"], "0.24.0": ["9.0.0"], "0.24.1": ["9.0.0"], "0.25.0": ["9.0.0"], "0.25.1": ["9.0.0"], "0.25.2": ["9.0.0"], "0.26.0": ["9.0.0"], "0.26.1": ["9.0.0"], "0.27.0": ["9.0.0"], "0.27.1": ["9.0.0"], "0.28.0": ["9.0.0"], "0.28.1": ["9.0.0"]}
```

Comment:
Are we supposed to see this? What does it mean?

Reply:
Good question, I see this in all my Bazel experiments so far. I have not enabled any debugging or other Bazel settings.

I assume you also see this when trying it on your Mac?

Reply:
Good question, I see this in all my Bazel experiments so far. I have not enabled any debugging or other Bazel settings.

I assume you also see this when trying it on your Mac?

Reply:
Indeed, I think PR #32 fixes this though.

Reply:
Ok will rerun it with that PR applied and update the documentation and this PR